ext/bg/background.html

```diff
@@ -12,7 +12,7 @@
         <link rel="icon" type="image/png" href="/mixed/img/icon128.png" sizes="128x128">
     </head>
     <body>
-        <div id="clipboard-paste-target" contenteditable="true"></div>
+        <textarea id="clipboard-paste-target" contenteditable="true"></textarea>
```

Comment:
The `contenteditable="true"` is probably redundant for a `<textarea>`

Reply:
Unclear why this is the case, but the cause seems to be the misformatted `<div>` tag.

https://github.com/kmltml/clipboard-inserter/blob/master/bg/index.html#L7
```html
<div id="paste-target" contenteditable="true" />
```

Changing this to a proper `<div>` tag makes it not work.
```html
<div id="paste-target" contenteditable="true"></div>
```

I've created an repository demonstrating the issue. If you want, you can test and confirm that you see the same issue (you have to open up the dev tools for the background page).
https://github.com/toasted-nutbread/firefox-clipboard-paste-bug

I've also created a Firefox bug report because it feels like a bug.
https://bugzilla.mozilla.org/show_bug.cgi?id=1603985